src/csharp/BUILD-INTEGRATION.md

```diff
@@ -105,6 +105,17 @@ button](https://stackoverflow.com/a/9770061). Click on it, and choose "Add as
 link". If you do not select this option, Visual Studio will copy files to the
 project directory instead.
 
+#### My .proto files have same filename in different folders
+
+By default Grpc.Tools compiles all .proto files into `obj` directory, flattening relative paths. For proto files with duplicated names it will cause following error `NETSDK1022 Duplicate 'Compile' items were included. [...]`. If you want to keep relative paths in your `obj` folder, define `OutputDir="$(Protobuf_OutputPath)%(RelativeDir)"`.
+
+```xml
+  <ItemGroup>
+    <Protobuf Include="Protos/v2/http.proto" OutputDir="$(Protobuf_OutputPath)%(RelativeDir)"  />
```

Comment:
What happens if the `.proto` file is outside of the project's directory structure (see previous section "My .proto files are in a directory outside the project")? I assume it's going to break.

If it breaks, is it fixable by using the `Link` attribute?

Reply:
Is there a way to make this work if I'm including multiple files with a wildcard? (e.g. `<Protobuf Include="**/*.proto" />`)

Reply:
Based on my experiment, using `<Protobuf Include="**/*.proto" OutputDir="$(Protobuf_OutputPath)%(RelativeDir)"/>` works just fine - can you please also try and confirm that is the case?

Reply:
I can confirm that wildcard was also working for me. I haven't included this in README because I have read in some docs that it is better to stay explicit.